sympy/core/tests/test_assumptions.py

```diff
@@ -726,7 +726,6 @@ def test_Mul_is_infinite():
     assert (x*i).is_finite is None
     assert (f*i).is_finite is False
     assert (x*f*i).is_finite is None
-    assert (z*i).is_finite is False
```

Comment:
Sounds reasonable.
It would be nice to find that test to be sure that it is present.
(And if it is not there, add it.)